src/smokeTest/java/uk/gov/hmcts/futurehearings/hmi/smoke/publication/PublicationApiSmokeTest.java

```diff
@@ -32,11 +32,11 @@ public void initialiseValues() throws Exception {
     }
 
     @Test
-    @DisplayName("Smoke Test to test the listings endpoint")
+    @DisplayName("Smoke Test to test the health endpoint")
     public void testPublicationHmiApiGet() {
         Response response = RestClient.makeGetRequest(getHeadersAsMap(), getAuthorizationToken(),
                 getRootContext());
 
-        assertEquals(HttpStatus.OK.value(), response.getStatusCode());
```

Comment:
Should this be kept and disable the test instead as this is what should be checked after the issues with P&I end point are fixed?

Reply:
wee should disable only when necessary. This way we will see the failure once integration is complete